### PR TITLE
Key Input handling changes

### PR DESCRIPTION
**Pull request by jakerist**
Changes key action from isDown to pressed so it doesn't keep opening the menu when the key is held down

### CODE REVIEW

**Comment by jakerist** [comment]
fixed the formatting issues

File: src/main/java/com/refinedmods/refinedstorage/screen/KeyInputListener.java [hunk 1]
```diff
@@ -64,25 +64,16 @@ public static void sendError(TranslatableComponent error) {
...
+        if (e.getAction() == 1 && Minecraft.getInstance().player != null) (
+            if (e.getKey() == RSKeyBindings.OPEN_WIRELESS_GRID.getKey()) {
+                findAndOpen(RSItems.WIRELESS_GRID.get(), RSItems.CREATIVE_WIRELESS_GRID.get());
+            } else if (e.getKey() == RSKeyBindings.OPEN_WIRELESS_FLUID_GRID.getKey()) {
+                findAndOpen(RSItems.WIRELESS_FLUID_GRID.get(), RSItems.CREATIVE_WIRELESS_FLUID_GRID.get());
+            } else if (e.getKey() == RSKeyBindings.OPEN_PORTABLE_GRID.getKey()) {
+                findAndOpen(RSItems.PORTABLE_GRID.get(), RSItems.CREATIVE_PORTABLE_GRID.get());
+            } else if (e.getKey() == RSKeyBindings.OPEN_WIRELESS_CRAFTING_MONITOR.getKey()) {
+                findAndOpen(RSItems.WIRELESS_CRAFTING_MONITOR.get(), RSItems.CREATIVE_WIRELESS_CRAFTING_MONITOR.get());
+            )
```
Comment (Darkere) [inline src/main/java/com/refinedmods/refinedstorage/screen/KeyInputListener.java:76]: wrong type of bracket?

Comment (jakerist) [reply src/main/java/com/refinedmods/refinedstorage/screen/KeyInputListener.java]: fixed

**Comment by Darkere** [comment]
Did you test this at all?

**Pull request closed without merging**